pkg/network/nettop/readme.md

```diff
@@ -3,3 +3,12 @@
 Nettop is a simple tool to ease testing the eBPF package.
 
 It loads the eBPF probes to watch for connections on the current host
+
+## Build
+
+```bash
+# At the root of the repository
+# First, build the system-probe to compile the eBPF programs.
+PATH=/usr/lib/llvm-12/bin:$PATH inv -e system-probe.build
```

Comment:
`inv -e system-probe.object-files` should just build the ebpf object files, without building the `system-probe` binary itself.

Reply:
The addition to `PATH` is likely not needed. The clang binary is downloaded from s3 before compile.